docs/source/changelog.md

```diff
@@ -7,6 +7,15 @@ list and release milestones.
 
 Release notes for the `space_packet_parser` library
 
+### v6.2.0 (Unreleased)
+
+- **BREAKING**: Migrated from Poetry to uv/hatchling for Python project management
+  - Build system now uses `hatchling` (PEP 517/518 compliant)
+  - Developer environment now uses `uv` for faster dependency resolution
+  - All Poetry-specific configuration removed from `pyproject.toml`
+  - Updated devcontainer, ReadTheDocs, and Conda configurations
+  - **Note**: This is only breaking for developers who were using Poetry directly. End users installing via pip or conda are unaffected.
```

Comment:
I see this note now. IMO, this is just worth a NOTE about what you've done here and not a breaking callout. If poetry is doing things right, this should be transparent to people who want to continue using poetry as well I think.

Reply:
Agreed. I'm also reducing it to a single note.